charts/helm-dep-installer/templates/job.yaml

```diff
@@ -76,5 +76,5 @@ spec:
             - name: CHART_VALUES
               value: {{ .Values.chartValues | toYaml | b64enc | quote }}
       restartPolicy: Never
-  backoffLimit: 10
+  backoffLimit: 2
```

Comment:
Have also reduced this to 2., lmk if you feel this can be more.

Reply:
default timeout is 300s from helm, that gives 10 mins for whatever is preventing the job from succeeding, so 2 is fine 👍